tonic/models/flo2d/utils.py

```diff
@@ -0,0 +1,140 @@
+from osgeo import osr
+import mpl_toolkits.basemap.pyproj as pyproj
+
+import numpy as np
+import pandas as pd
+
+from scipy import stats
+import quantities as pq
+from scipy.spatial import cKDTree
+
+# Default projection from Alan Hamlet 2015-06-29
+default_proj = pyproj.Proj('+proj=lcc +lat_1=47.5 +lat_2=48.73333333333333 '
+                           '+lat_0=47'' +lon_0=-120.8333333333333 '
+                           '+x_0=609601.2192024385 +y_0=0 +datum=NAD83 '
+                           '+units=us-ft')
+
+
+def esriprj2standards(shapeprj_path, kind='proj4'):
+    '''Return string representation of projection in shapeprj_path.
+
+    Parameters
+    ----------
+    shapeprj_path : str
+        Filepath to ESRI style projection file.
+    kind : str {'prj', 'wkt', 'proj4', 'epsg'}
+        Projection standard.
```

Comment:
Maybe `Projection standard output type` just to clarify that it's the output format you're specifying